WooCommerce/Classes/Extensions/WordPressComSiteInfo+Woo.swift

```diff
@@ -5,12 +5,6 @@ extension WordPressComSiteInfo {
     /// Encapsulates the rules that declare a WordPress site as having a valid
     /// Jetpack installation
     var hasValidJetpack: Bool {
-        if ServiceLocator.featureFlagService.isFeatureFlagEnabled(.jetpackConnectionPackageSupport) {
-            return isJetpackConnected
-        } else {
-            return hasJetpack &&
-                isJetpackConnected &&
-                isJetpackActive
-        }
+        return isJetpackConnected
```

Comment:
Now that the feature flag is gone, do we need this encapsulation? Or should we remove this extension entirely and get the check from `WordPressComSiteInfo.isJetpackConnected` directly?

For context of why we only need this check, on https://github.com/woocommerce/woocommerce-ios/pull/5474 

> Because of inaccurate values in isJetpackActive and hasJetpack (p91TBi-6lK-p2), we are going to just use isJetpackConnected for determining if a site is connected to Jetpack (either via JCP or Jetpack-the-plugin).

Reply:
I'd say removing this extension and replacing the current usage with `isJetpackConnected`, since there's no extra logic in this extension anymore

Reply:
Thanks for confirming. Replaced here: 38036aaafb2e84b20f1f893810253e686e9c97d6